doc/nrf/gs_installing.rst

```diff
@@ -229,7 +229,7 @@ To clone the repositories, complete the following steps:
 #. Determine what revision of the |NCS| you want to work with.
    The recommended way is to work with a specific release.
 
-   * To work with a specific release, the revision is the corresponding tag (for example, ``v1.3.0``).
+   * To work with a specific release, the revision is the corresponding tag (for example, ``v1.3.2``).
```

Comment:
Can't we leave this unchanged? It's a an example. In fact I think it should be set back to 1.2.0 to match the development tag example in the bullet point below.